libbeat/docs/config-file-format.asciidoc

```diff
@@ -48,7 +48,7 @@ form is somewhat similar to JSON using `{}` for dictionaries and `[]` for lists:
 
 ["source","yaml",subs="attributes"]
 ------------------------------------------------------------------------------
-person: \{name: "John Doe", age: 34, country: "Canada"}
+person: {name: "John Doe", age: 34, country: "Canada"}
 colors: ["Red", "Green", "Blue"]
```

Comment:
@nik9000 Sorry I wanted to rebase so force-pushed over my comments. This changes breaks the asciidoc build, but is required to get the formatting right in asciidoctor. Let me know if you have a different solution.

The error in asciidoc is:

```
12:20:15 INFO:build_docs:asciidoc: WARNING: config-file-format.asciidoc: line 52: illegal system attribute name: name
```

Reply:
actually I asked this question in a different PR, so no need to answer here...just answer in the original.